### DISCUSSION TITLE
How to install a global prettier plugin and use it in VS Code?

### DISCUSSION THREAD

**Discussion by kud** [prettier/prettier-vscode#2874] [Q&A]
I'd like to add the plugin for "ini" file and use it for any project in VS Code. Is there a way to do this?

**Comment by wegotoeleven** [comment]
Same! I'd like to install [`prettier-plugin-sh`](https://github.com/un-ts/prettier/tree/master/packages/sh) and use it with prettier in vscode, but it's really not obvious how to do so!
